cmd/dockerd/daemon.go

```diff
@@ -287,24 +239,39 @@ func (cli *DaemonCli) start() (err error) {
 		"graphdriver": d.GraphDriverName(),
 	}).Info("Docker daemon")
 
-	cli.initMiddlewares(api, serverConfig)
-	initRouter(api, d, c)
+	// setup the default routes
+	defaultRoutes := getRouters(d, c)
+	api.InitRouter(utils.IsDebugEnabled(), defaultRoutes...)
+
+	// setup the introspections routes without cluster capabilities
```

Comment:
for many reasons.  swarm needs more attention or else we create a backdoor and undo many of the security measures that were built in.  It is also inconsistent if you are run on a manager/worker.